perception/vis/vis.py

```diff
@@ -2,21 +2,26 @@
 from pathlib import Path
 from FrameWrapper import FrameWrapper
 import cv2 as cv
-from yukiVisualizer import Visualizer
+from window_builder import Visualizer
+import cProfile as cp
+import pstats
 
 # import TestTasks.testAlgo
 # Collect available datasets
+"""
```

Comment:
Why do we have random commented-out code sprinkled around? They should be removed or added as optional features

Reply:
I removed these unnecessary comments